### ISSUE TITLE
Phone Contacts

### ISSUE THREAD

**Issue by bflanagin** [OpenSeedINC/CafeSync#59]
the ability to add all the users contacts and scan for other CafeSync users would be a real boone for the system.

**Comment by ryanleesipes** [comment]
Yes, Telegram does this and it is awesome. However, Telegram also seems to base its entire system off of phone numbers (or at least it did, not sure how much the case this is now).
